docs/sidebars.js

```diff
@@ -5,30 +5,34 @@ module.exports = {
       'Flashbots Core': [
         'flashbots-core/overview',
         {
-          "for searchers": [
+          "For searchers": [
```

Comment:
same same comment about keeping everything lowercase vs not, no strong feelings there personally